maas/resource_maas_boot_source_selection.go

```diff
@@ -59,15 +64,18 @@ func resourceMAASBootSourceSelection() *schema.Resource {
 				Description: "The specific release of the operating system for this selection.",
 			},
 			"subarches": {
-				Type:     schema.TypeSet,
-				Elem:     &schema.Schema{Type: schema.TypeString},
-				Optional: true,
-				DefaultFunc: func() (any, error) {
-					return []any{"*"}, nil
-				},
-				Description: "The list of subarches for this selection.",
+				Type:        schema.TypeSet,
+				Elem:        &schema.Schema{Type: schema.TypeString},
+				Optional:    true,
+				Computed:    true,
+				Description: "The list of subarches for this selection. Default is: `[\"*\"]`",
 			},
 		},
+		Timeouts: &schema.ResourceTimeout{
+			Create: schema.DefaultTimeout(30 * time.Minute),
+			Update: schema.DefaultTimeout(30 * time.Minute),
+			Delete: schema.DefaultTimeout(30 * time.Minute),
```

Comment:
we may want to revisit these timeouts and make them user definable in future, for now i think 30 min default is fair though

Reply:
By putting them in the schema, they are user definable. The 30 minutes here means: "If the user didn't provide any value, default to 30 minutes"

Reply:
Oh! my mistake, thanks!